templates/kafka-connect/kafka-connect.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.
Same question as above, it's not a template btw.